node-core/components/backend.go

```diff
@@ -52,5 +54,6 @@ func ProvideStorageBackend(
 		in.BeaconStore,
 		in.DepositStore,
 		in.BlockStore,
+		in.Logger,
```

Comment:
consider `in.Logger.With("backend", "storage")` to align this with what we do with other services.